src/api_examples.py

```
@@ -0,0 +1,188 @@
+                    "cpu_usage": 45.2,
+                    "memory_usage": 67.8,
+                    "disk_usage": 23.1,
+                    "request_count": 1250,
+                    "error_count": 5
+                },
+                "curl_command": 'curl -X GET "http://localhost:5000/api/health/"'
+            }
+        }
+
+    @api.marshal_with(example_response)
+    def get(self, example_type: str = None):
```

Ruff (0.12.2)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)